examples/jsm/objects/Sky.js

```diff
@@ -28,10 +28,10 @@ class Sky extends Mesh {
 		const shader = Sky.SkyShader;
 
 		const material = new ShaderMaterial( {
-			name: 'SkyShader',
-			fragmentShader: shader.fragmentShader,
-			vertexShader: shader.vertexShader,
+			name: 'shader.name',
```

Comment:
I guess the `'` signs should be removed.

Reply:
fixed!